Lab6

yinyangcolby opened the pull request:
I am once again sorry for not having anything for you  to review, I am traveling back home from vacation and will be able to give you something very soon. But at the least I'd want for you to have something to turn in.

yinyangcolby committed:
Lab 4

Lab 4 submission

yinyangcolby committed:
Lab 3 Updated So Forms Work

Forums works now.

yinyangcolby committed:
Pages updated with correct navbar

yinyangcolby committed:
Forums has bootstrap

yinyangcolby committed:
Unit testing and quiz added

yinyangcolby committed:
Updated xUnit version

xUnit version 2.4.1 was installed, causing an error in the tests. This issue is fixed and now all tests are working.

readyR0840 commented:
| **Criteria**                                                 | Check |
| ------------------------------------------------------------ | ----- |
| Does it compile and run without errors                       | [ Y]   |
|                                                              |       |
| Do all the pages load correctly?                             | [ Y]   |
|                                                              |       |
| Do all the links, buttons or other UI elements work correctly? | [Y]  |
|                                                              |       |
| Does the web app provide a good user experience (UX)?        | [Y]  |
|                                                              |       |
| Does the naming and structure conform to MVC conventions?<br />(Names of controllers, controller methods, views, etc.) | [Yes]    |
|                                                              |       |
| Does the style conform to C# coding conventions?             | [Almost] The razor code looks good, but now the controller still uses viewbag. The best way is probably to pass in model to the view.   |
|                                                              |       |
| Do the design and implementation conform to best practices?  | [N]   | 
|                                                              |       |
| Does the solution meet all the requirements in the lab instructions? | [ N]  Does not use migration to access a dbcontext object                    |
|                                                              |       |
| Summary Comments:                                            |   I think you should make the list you have a model, and make it get a db context entity field from a database.  |
|                                                              |       |

readyR0840 commented:
Sorry, I think the requirement of this code review wants me to upload my copy of the code review. I will try to come back and comment anyway when it is done.

yinyangcolby committed:
EFCore and Database

Nuget packages added, forums has been refactored to include database. You can leave feedback and it will remain there.
